src/hooks/useCharactersApi.ts

```diff
@@ -63,7 +65,7 @@ const useCharactersApi = (searchParams) => {
   }, [characters]);
 
   const handleNextPage = () => {
-    setSize((prevSize) => prevSize + 1);
+    setSize((prevSize: number) => prevSize + 1);
```

Comment:
[pp] `prevSize` type is already known from the `useSWRInfinite` hook
```suggestion
    setSize((prevSize) => prevSize + 1);
```